packages/vite/src/modes/svelte-scoped.ts

```diff
@@ -10,6 +10,8 @@ export function SvelteScopedPlugin({ uno, ready }: UnocssPluginContext): Plugin
     const { css } = await uno.generate(code)
     if (!css)
       return null
+    if (code.match(/<style[^>]*>[\s\S]*?<\/style\s*>/))
+      return code.replace(/(<style[^>]*>)/, `$1${css}`)
```

Comment:
Should we add it to the end?

Reply:
Either way. My thought is adding at the beginning should allow the css in the style element override the unocss-generated ones, just as the recent PR https://github.com/antfu/unocss/pull/458

Reply:
Anyway, let's have it until someone complain about it 👀